Emby.Naming/Video/Format3DParser.cs

```diff
@@ -15,10 +15,12 @@ public Format3DParser(NamingOptions options)
 
         public Format3DResult Parse(string path)
         {
-            var delimeters = _options.VideoFlagDelimiters.ToList();
-            delimeters.Add(' ');
+            int oldLen = _options.VideoFlagDelimiters.Length;
+            var delimeters = new char[oldLen + 1];
+            _options.VideoFlagDelimiters.CopyTo(delimeters, 0);
+            delimeters[oldLen] = ' ';
```

Comment:
Is the efficiency worth it?

Reply:
I don't have any benchmarks atm, but considering that the old code would allocate at least 2 times (3 times in worst case scenarios) while the new code only allocates once, I'd say it's worth it. (Depending on how hard you want to micro-optimize 😉)